### PR TITLE
feat(Viper 92,94): add the payment interface, choose payment method

### PR DESCRIPTION
**Pull request by tran-duy2**

### CODE REVIEW

File: src/pages/shipping.tsx [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+            <select
+              value={shippingInput.province}
+              onChange={(e) => setShippingInput({ ...shippingInput, province: e.target.value })}
+              className="p-2 border rounded-md"
+            >
+              <option>Province</option>
+              <option>KonTum</option>
+              <option>Da Nang</option>
+            </select>
+            <select
```
Comment (thienbuidev) [inline src/pages/shipping.tsx:104]: Try using real data city

File: src/pages/shipping.tsx [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+              value={shippingInput.postalCode}
+              onChange={(e) => setShippingInput({ ...shippingInput, postalCode: e.target.value })}
+              className="p-2 border rounded-md"
+            />
+            <select
+              value={shippingInput.province}
+              onChange={(e) => setShippingInput({ ...shippingInput, province: e.target.value })}
+              className="p-2 border rounded-md"
+            >
+              <option>Province</option>
```
Comment (thienbuidev) [inline src/pages/shipping.tsx:100]: try using real data

File: src/pages/shipping.tsx [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+        <span>Payment</span>
+      </div>
+
+      <div className="flex flex-col md:flex-row p-4 bg-gray-50 min-h-screen">
+        <div className="flex-1 bg-white p-6 shadow-lg rounded-md">
+          <h2 className="text-lg font-semibold mb-4">Customer Information</h2>
+
+          <div className="mb-6 border-t pt-4">
+            <label className="block text-sm font-medium mb-2">Contact</label>
+            <input
```
Comment (thienbuidev) [inline src/pages/shipping.tsx:45]: Using Form.Item > Input of Ant Design.

File: src/pages/payment.tsx [hunk 4]
```diff
@@ -0,0 +1,343 @@
...
+                    <div className="mt-4 flex justify-center">
+                      <img
+                        src={QRVNPay}
+                        alt="Spiced Mint Candleaf"
+                        className="w-40 h-40 object-contain"
+                      />
+                    </div>
+                  )}
+
+                  {cardFormVisible && (
```
Comment (thienbuidev) [inline src/pages/payment.tsx:197]: try seperate new component

File: src/pages/payment.tsx [hunk 5]
```diff
@@ -0,0 +1,343 @@
...
+                      value={method}
+                      checked={paymentMethod === method}
+                      onChange={(e) => handlePaymentMethodChange(e.target.value)}
+                    />
+                    <span>{method}</span>
+                  </label>
+                ))}
+              </div>
+
+              {showPaymentDetails && (
```
Comment (thienbuidev) [inline src/pages/payment.tsx:163]: try to new component

**Review by thienbuidev — CHANGES_REQUESTED** [review]

**Review by bxthien — APPROVED** [review]

**Review by bxthien — APPROVED** [review]

**Review by bxthien — APPROVED** [review]

**Review by thienbuidev — APPROVED** [review]

**Pull request merged**